sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonHighlighterTest.java

```diff
@@ -211,14 +212,20 @@ void number() {
 
   @Test
   void highlightingNotebooks() {
-    String pythonContent = "def foo():\n    pass\na = \"test\" # comment\n# test \\n \\\\n test\nb = 3J\n#SONAR_PYTHON_NOTEBOOK_CELL_DELIMITER";
+    String pythonContent = """
```

Comment:
Maybe in the highlighter we could add a `\t` to one of the test. To be sure the impact of the changes you made are propagated here.